src/buttons/buttons.service.js

```diff
@@ -96,6 +88,25 @@ export default class ButtonService {
   static strcmp(string1, string2) {
     if (string1.toString() < string2.toString()) return -1;
     if (string1.toString() > string2.toString()) return 1;
+
     return 0;
   }
+
+  /**
+   * Get the current date in format yyyy-mm-dd h:m:s
+   *
+   * @return string
+   */
+  static getCurrentDate() {
```

Comment:
There is a function in JS: Date.toLocaleString()
This will automatically generate the date in the format the user is used to. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString